metricbeat/module/golang/_meta/config.yml

```diff
@@ -0,0 +1,8 @@
+- module: golang
+  metricsets: ["expvar","heap"]
+  enabled: true
+  period: 10s
+  hosts: ["localhost:6060"]
+  metrics_path: "/debug/vars"
```

Comment:
We should probably have this under `expvar.path` and `heap.path` as these option do not necessarly exist or are different in other metricsets we add in the future to golang.